### PR TITLE
Worked on css

### PR DESCRIPTION
**Pull request by bisht-richa**
## Changes
- Added some new classes in HAML files and in css file.
- Improved the appearance of the hackweek project page (right side panel). 
- Apply css to all Header (H1, H2..), p and ul/ui tag.
- Improved the appearance of right side panel.
- Moved News, About, pics and credit links from footer to header section.

## Screen shot
screen shot 2016-06-20 at 14 56 22
screen shot 2016-06-20 at 15 03 21

### CODE REVIEW

**Comment by kobliha** [comment]
@hennevogel This patch fixes the appearance of a project details so now you know what's a headline and what's a project description. The other changes are somehow required to promote the other content. It still might need some cleanup, but after @bisht-richa 's changes it's much more understandable and I recommend to merge it (please) ;) LGTM from my POV

File: app/assets/stylesheets/hackweek.css.scss [hunk 1]
```diff
@@ -10,8 +10,24 @@ body {
   background-image: url("data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHdpZHRoPSI1IiBoZWlnaHQ9IjUiPgo8cmVjdCB3aWR0aD0iNSIgaGVpZ2h0PSI1IiBmaWxsPSIjZmZmIj48L3JlY3Q+CjxyZWN0IHdpZHRoPSIxIiBoZWlnaHQ9IjEiIGZpbGw9IiNjY2MiPjwvcmVjdD4KPC9zdmc+");
 }
 
+h4.project-list-item {
+  font-size: 24px;
```
Comment (hennevogel) [inline app/assets/stylesheets/hackweek.css.scss]: A fixed font size looks ugly on mobile

File: app/assets/stylesheets/hackweek.css.scss [hunk 2]
```diff
@@ -45,15 +61,131 @@ footer {
...
-    position: relative;
-    content:"";
-    height:70px;
-    margin:-70px 0 0;
-    z-index: -1;
+  display:block;
+  position: relative;
+  content:"";
+  /* height:70px;
+   margin:-70px 0 0;*/
```
Comment (hennevogel) [inline app/assets/stylesheets/hackweek.css.scss]: Please do not comment out code. Delete it if it's not needed.

File: app/assets/stylesheets/hackweek.css.scss [hunk 3]
```diff
@@ -10,8 +10,24 @@ body {
...
+
+h4.project-list-item img {
+  width: 26px;
+  margin-right: 10px;
+  max-height: 30px;
+}
+
+p {
+    margin: 15px 0 0 0;
+    font-size: 16px;
```
Comment (hennevogel) [inline app/assets/stylesheets/hackweek.css.scss]: Please do not just change HTML tags layout. You'll never know where `<p>` will be used

File: app/assets/stylesheets/hackweek.css.scss [hunk 4]
```diff
@@ -45,15 +61,131 @@ footer {
...
+   margin:-70px 0 0;*/
+  z-index: -1;
 }
 
 #loading-spinner {
   text-align: center;
   vertical-align: bottom;
 }
+
+.well {
```
Comment (hennevogel) [inline app/assets/stylesheets/hackweek.css.scss]: Please do not change the layout of bootstrap components. I guess you want to target a specific well, add another class please.

File: app/assets/stylesheets/hackweek.css.scss [hunk 5]
```diff
@@ -45,15 +61,131 @@ footer {
...
+  margin-top: 5px;
+  margin-bottom: 15px;
+}
+small {
```
Comment (hennevogel) [inline app/assets/stylesheets/hackweek.css.scss]: I believe what I said about `<p>` is also true for `<small>` :-)

**Comment by kobliha** [comment]
@hennevogel Could you PLS review again and eventually merge and deploy? Thx

**Comment by hennevogel** [comment]
@bisht-richa please remove the bit that moves the footer links to the top. I don't like it at all.

**Comment by bisht-richa** [comment]
I have moved links from header to bottom. Please check.

**Comment by kobliha** [comment]
@hennevogel PLS merge and deploy, thanks in advance.

**Pull request merged**